# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vortexge/zlib.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- VortexGE/buildScript/configure/zlib-functions.inc.vanilla	2007-01-01 18:38:10.000000000 +0000
+++ VortexGE/buildScript/configure/zlib-functions.inc	2007-01-01 18:38:57.000000000 +0000
@@ -30,7 +28,7 @@
     fi
   done
 
-  for __Loc__ in $USRLibLoc $STDLibLoc; do
+  for __Loc__ in $USRLibLoc $STDLibLoc /lib* ; do
     if [[ -e $__Loc__/libz.so ]]; then
       export $2=$__Loc__
       break;
